docsrc/source/dipolar_guide_modeling.rst

```diff
@@ -3,41 +3,55 @@
 Modeling
 =========================
 
-DeerLab provides a very flexible framework to model dipolar signals originating from any dipolar EPR spectroscopy experiments. Choosing a model that properly describes your sample and experiment is of paramount importance. The DeerLab function ``dipolarmodel`` already defines the core model structure based on dipolar pathways, with the following components to be chosen:     
+DeerLab provides a very flexible framework to simulate/model signals from a wide range dipolar EPR spectroscopy experiments. Use the DeerLab function ``dipolarmodel`` to define the model.
 
-* **Distance range**: Also called the interspin distance axis, is the range of distances where the distribution is defined. 
+    Vmodel = dl.dipolarmodel(t,r,Pmodel,Bmodel,experiment)
 
-* **Distribution model**: Describes the intra-molecular distance distribution in either a parametric (e.g. a Gaussian distribution) or a non-parametric way. 
+The following components need to be chosen:
 
-* **Background model**: Describes the dipolar background signal arising from the inter-molecular contributions. 
+* **Time range** ``t``: This is the range of dipolar evolution times for the desired signal.
 
-* **Number of pathways**: Sets the number of dipolar pathways contributing to the dipolar signal.
+* **Distance range** ``r``: This is the range of intra-molecular spin-spin distances where the distribution is defined.
 
-For each of these four components, a choice needs to be made: 
+* **Distribution model** ``Pmodel``: Describes the intra-molecular distance distribution in either a parametric form (e.g. a Gaussian distribution) or a non-parametric form (as a histogram). 
+
+* **Background model** ``Bmodel``: Describes the dipolar background signal arising from the inter-molecular spin pairs. 
+
+* **Type of experiment** ``experiment``: This sets the number of dipolar pathways contributing to the dipolar signal.
+
+For each of these components, a choice needs to be made: 
+
+Choosing a time range
+*************************
+
+The time range depends on the expected range of distances, but is typically in the range of 0.5 to several microseconds. To construct a time vector, use the ``linspace`` function from NumPy: ::
```

Comment:
I am unsure of this wording here. When creating the model you have already got your data, so you aren't so much choosing your time range more inputting it. I also wouldn't say that the time range is dependent on the distances, more the range of extractable distances is dependent upon the time axis. 

Additionally most people use a tau1 of 0.4us for 4pDEER, so the range starts before 0.5 us.